addons/osfstorage/tests/test_views.py

```diff
@@ -1548,7 +1548,7 @@ def test_download_file_with_token(self, mock_get_client):
             )
         )
 
-        download_url = base_url.format(file.get_guid()._id)
+        download_url = base_url.format(file._id)
```

Comment:
I think the issue is that the file doesn't have a GUID generated yet. So rather than using the file ID, we can use the create the GUID when obtaining it.

```python
download_url = base_url.format(file.get_guid(create=True)._id)
```

Reply:
Oh, forgot to mention, your fix works (i.e. passing the CI) since we can access the file by file ID. However, I think are testing the view using GUID here.